### PR TITLE
vict-4

### LINKED ISSUE #9: vict-4: [Basic Create] - Create layout for creating cases (8h)
Create a layout for creating new cases page:
- It is accessible by path **/cases/new**
- It uses layout from theme: [https://bootstrapmade.com/demo/NiceAdmin/](https://bootstrapmade.com/demo/NiceAdmin/):
- Input of data:
    - **card** class is used
    - Header in the card: "Створити нову справу"
    - For fields "**CURRENT_STAGE**", "**STAGE_DEADLINE**", "**JUDGE**", "**NUMBER**", "**OPPONENT**", "**SUBJECT**", "**CLIENT**", "**NOTE**" there are text inputs
    - For field "**DONE_WORK**" there is a text area input
- In the bottom there are 2 buttons:
    - "Назад" - redirect to listing page
    - "Створити" - redirect to listing page too

Acceptance criteria:
- layout of page is created
- page is accessible from correct path
- data is not sent anywhere controller yet

### PR DESCRIPTION
**Pull request by Ratchet34702**
Create a layout for creating new cases page
Add NewCase controller
Move scripts into main.script.rocker.html

### CODE REVIEW

**Comment by Ratchet34702** [comment]
worked 5 hours

File: src/main/resources/views/mainScript.rocker.html
Comment (makskostyshen) [inline src/main/resources/views/mainScript.rocker.html:1]: It's good you've created this file. But you do not use it anywhere) Now we do not have logic with possible JS involved, but when it's added, we'll have problems. Let's create a main rocker html file that will have everyting except <main> content. This file will have 2 arguments: page name and rocker body. Look at rocker body here: https://github.com/fizzed/rocker/blob/master/docs/SYNTAX.md

**Review by makskostyshen — COMMENTED** [review]
Fix is needed

**Review by makskostyshen — APPROVED** [review]
approved

**Pull request merged**